src/main/java/at/ac/tuwien/damap/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -297,7 +297,7 @@ public void contributorInformation(){
     public void costInformation(){
         String costs = "";
         if (dmp.getCostsExist() != null) {
-            if (dmp.getCostsExist().booleanValue()) {
+            if (dmp.getCostsExist()) {
                 costs = loadResourceService.loadVariableFromResource(prop, "costs.avail");
             }
             else {
```

Comment:
How about we simplify this to
```java
        if (Boolean.TRUE.equals(dmp.getCostsExist()))
```

Then we can get rid of the nested conditional

Reply:
Good catch!